src/ontology/edits/oeo-physical.omn

```diff
@@ -1170,37 +1243,34 @@ Class: Graph
         <http://purl.obolibrary.org/obo/BFO_0000024>
     
     
-Class: GreenhouseGas
+Class: GreenhouseEffectDisposition
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "A greenhouse gas is a gas that absorbs and emits radiant energy within the thermal infrared range. Greenhouse gases cause the greenhouse effect.",
-        <http://purl.obolibrary.org/obo/IAO_0000119> "https://en.wikipedia.org/w/index.php?title=Greenhouse_gas&oldid=880592155"
+        <http://purl.obolibrary.org/obo/IAO_0000115> "The greenhouse effect disposition is the disposition of a gas to contribute to the greenhouse effect, when it is emitted into the atmosphere via an emission.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue #56, pull request #234"
     
     SubClassOf: 
-        Pollutant
+        <http://purl.obolibrary.org/obo/BFO_0000016>
     
     
-Class: Grid
+Class: GreenhousegasEmission
```

Comment:
If we include the class `GreenhousegasEmission` we should make `CarbonDioxideEmission` a subclass of it. Currently it is also a direct subclass of `Emission`.

Reply:
changed it